.github/codecov.yml

```diff
@@ -0,0 +1,15 @@
+comment: false
+
+github_checks:
+  annotations: false
+
+coverage:
+  status:
+    patch: false
+    # project will give us the diff in the total code coverage between a commit
+    # and its parent
+    project:
+      default:
+        target: auto
+        threshold: "15%"
+    changes: false
```

Comment:
These were copied from what we had in moby/moby and docker/cli. Not sure if they're all needed, but at least basic functionality works. We could enable it leaving comments, but they are sometimes a bit noisy, so I left that off for now